lang/python/pyproject-metadata/Makefile

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (C) 2006-2011 OpenWrt.org
+# Copyright (C) 2015-2016, 2018-2021, 2023 Jeffery To
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=pyproject-metadata
+PKG_VERSION:=0.7.1
+PKG_RELEASE:=1
+
+PYPI_NAME:=pyproject-metadata
+PKG_HASH:=0a94f18b108b9b21f3a26a3d541f056c34edcb17dc872a144a15618fed7aef67
+
+PKG_MAINTAINER:=Jeffery To <jeffery.to@gmail.com>

```

Comment:
Is @jefferyto aware of it?

Reply:
done
maintainer:me

Reply:
I doubt it that it is done, though

Reply:
can you see my current changes. what should be changed